src/gnosisSafe/contracts/safeContracts.js

```diff
@@ -0,0 +1,56 @@
+export const SENTINEL_ADDRESS = "0x0000000000000000000000000000000000000001";
+export const MULTI_SEND_ADDRESS = "0x8d29be29923b68abfdd21e541b9374737b49cdad";
+export const SAFE_MASTER_COPY_ADDRESS =
+  "0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F";
+export const DEFAULT_FALLBACK_HANDLER_ADDRESS =
+  "0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44";
+export const SAFE_MASTER_COPY_ADDRESS_V10 =
```

Comment:
We are not using `SENTINEL_ADDRESS`, `MULTI_SEND_ADDRESS`,  `SAFE_MASTER_COPY_ADDRESS `, `DEFAULT_FALLBACK_HANDLER_ADDRESS `, `SAFE_MASTER_COPY_ADDRESS_V10 ` addresses anywhere, right?

Reply:
If so, can we remove it?